### PR TITLE
librz: add some initial documentation

### PR DESCRIPTION
**Pull request by ret2libc**
Partially addresses https://github.com/rizinorg/rizin/issues/286

### CODE REVIEW

File: librz/util/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# RzUtil
+
+This library contains all common utilities and functions that are used by other
+modules in the Rizin framework, like string manipulation, data structures
+(lists, vectors, trees, etc.), JSON parsing/printing, and others.
+
+## What can I expect here?
+- Functions that are very common and not specific to other modules
```
Comment (XVilka) [inline librz/util/README.md]: Numbers/math

**Review by XVilka — CHANGES_REQUESTED** [review]

**Comment by ret2libc** [comment]
@wargio @thestr4ng3r @XVilka please review if the two files i added for now are ok. If so I think we can merge so that also others, if they want, can start filling the docs.

File: librz/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Rizin Libs (librz)
+
+The Rizin framework is composed of several pieces that work together to provide
+a nice reverse-engineering framework useful for a variety of use-cases and
```
Comment (XVilka) [inline librz/README.md]: Remove `a nice`.

File: librz/io/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+the *cache* concept in RzIO. The cache provides a layer on top of `RzIOMap` to
+write/read data temporarily.
+
+## What can I expect here?
+- I/O operations, like read from address, write to address, seek, resize, etc.
+- Functions to open new files/resources
+- Functions to map parts of a RzIODesc into the Rizin space
+- Functions to manipulate maps and opened descriptors
+- Functions to deal with the cache
+- RzIO Plugins to perform I/O operations on various kinds of resources
```
Comment (XVilka) [inline librz/io/README.md]: Put `RzIO` into backticks

File: librz/util/README.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# RzUtil
+
+This library contains all common utilities and functions that are used by other
+modules in the Rizin framework, like string manipulation, data structures
+(lists, vectors, trees, etc.), JSON parsing/printing, and others.
+
+## What can I expect here?
```
Comment (XVilka) [inline librz/util/README.md:7]: Add floating point, SDB. Also worth mentioning providing OS/platform-independent common abstractions.

**Review by XVilka — CHANGES_REQUESTED** [review]
Yes, once feedback is addressed I think we can merge. Step by step.

**Comment by ret2libc** [comment]
@XVilka feedback addressed.

**Review by XVilka — APPROVED** [review]

**Pull request merged**